### PR TITLE
Rewrite of the Sensitive Data article

### PR DESCRIPTION
**Pull request by sofietoft**
Restructured the article so the flow is better and makes more sense.

### CODE REVIEW

File: Reference/Security/sensitive-data.md [hunk 1]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
 
 # Sensitive data
 
-Marking fields as sensitive will hide the data in those fields for backoffice users that have no business viewing personal data of members.
+Marking fields and properties on member data as sensitive will hide the data in those fields for backoffice users that are not privy to the data.
 
-In order to start marking fields as sensitive, you can add the users who need access to this data to the "Sensitive data" user group. In the User management sections go to Groups and choose the Sensitive data group.
+In this article you will get an overview on how you can grant and/or deny your users access to sensitive data as well as how to mark data as sensitive.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:10]:
```suggestion
In this article, you will get an overview of how you can grant and/or deny your users access to sensitive data as well as how to mark data as sensitive.
```

File: Reference/Security/sensitive-data.md [hunk 2]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+
+## Grant or deny access to sensitive data
+
+Every new Umbraco installation ships with a default set of User Groups. One them is the **Sensitive data User Group**. In order to give users in the backoffice access to view and work with sensitive data, they need to be part of the Sensitive data User Group.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:14]:
```suggestion
Every new Umbraco installation ships with a default set of User Groups. One of them is the **Sensitive data User Group**. To give users in the backoffice access to view and work with sensitive data, they need to be part of the Sensitive data User Group.
```

File: Reference/Security/sensitive-data.md [hunk 3]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+
+Any users who are not part of the Sensitive data User Group, will not be able to see the data in the properties that are marked as sensitive. Instead they will will see a generic message: "*This value is hidden. If you need access to view this value please contact your website administrator.*"
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:16]:
```suggestion
Any users who are not part of the Sensitive data User Group, will not be able to see the data in the properties that are marked as sensitive. Instead, they will see a generic message: "*This value is hidden. If you need access to view this value please contact your website administrator.*"
```

File: Reference/Security/sensitive-data.md [hunk 4]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+
+![Sensitive data hidden](images/sensitive-data-hidden-v8.png)
+
+While not part of the Sensitive data User Group it is also not possible to export members or member data.
+
+Follow these steps in order to grant a user access to sensitive data.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:22]:
```suggestion
Follow these steps in order to grant a user access to sensitive data:
```

File: Reference/Security/sensitive-data.md [hunk 5]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+
+* Navigate to the **User** section in the Umbraco backoffice.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:24]:
```suggestion
* Navigate to the **Users** section in the Umbraco backoffice.
```

File: Reference/Security/sensitive-data.md [hunk 6]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+* Select the **Groups** menu in the top-right corner.
+* Choose the **Sensitive data** group.
 
 ![Sensitive data user group](images/sensitive-data-user-group-v8.png)
 
-From there on, add the users who need access to this data to this group.
+* Click **Add** in the User box on the right.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:30]:
```suggestion
* Click **Add** in the **Users** box on the right.
```

File: Reference/Security/sensitive-data.md [hunk 7]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+Any users who are not part of the Sensitive data User Group, will not be able to see the data in the properties that are marked as sensitive. Instead they will will see a generic message: "*This value is hidden. If you need access to view this value please contact your website administrator.*"
+
+![Sensitive data hidden](images/sensitive-data-hidden-v8.png)
+
+While not part of the Sensitive data User Group it is also not possible to export members or member data.
+
+Follow these steps in order to grant a user access to sensitive data.
+
+* Navigate to the **User** section in the Umbraco backoffice.
+* Select the **Groups** menu in the top-right corner.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:25]: To access the Groups menu, we have to click on users in the left pane and then it shows up. 
image

Maybe add or update this step?

File: Reference/Security/sensitive-data.md [hunk 8]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+* Choose the **Sensitive data** group.
 
 ![Sensitive data user group](images/sensitive-data-user-group-v8.png)
 
-From there on, add the users who need access to this data to this group.
+* Click **Add** in the User box on the right.
+* Select the sers you want to give access to the sensitive data.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:31]:
```suggestion
* Select the users you want to give access to the sensitive data.
```

File: Reference/Security/sensitive-data.md [hunk 9]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+* Click **Submit**.
+* **Save** the User Group.
 
-![Update member type](images/update-member-type-v8.png)
+The users you have added to the Sensitive data User Group will now be able to
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:35]:
```suggestion
The users you have added to the Sensitive data User Group will now be able to:
```

File: Reference/Security/sensitive-data.md [hunk 10]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
 
-When a user in the backoffice does not have access to this data they get told so:
+* See member data that has been marked as sensitive,
+* Mark data and properties on Member Types as sensitive and
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:38]:
```suggestion
* Mark data and properties on Member Types as sensitive, and
```

File: Reference/Security/sensitive-data.md [hunk 11]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+* Export members and member data.
+
+## Marking data as sensitive
+
+Once your user is added to the Sensitive data User Group, you have access to add and configure member properties containing sensitive data.
+
+* Navigate to the **Settings** section in the Umbraco backoffice.
+* Double-click on **Member Types** in the Settings tree.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:46]:
```suggestion
* Open the **Member Types** in the **Settings** tree.
```

File: Reference/Security/sensitive-data.md [hunk 12]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+* Select the Member Type you wish to edit.
+* **Add** a property, or configure an existing property.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:48]:
```suggestion
* **Add** a property or configure an existing property.
```

File: Reference/Security/sensitive-data.md [hunk 13]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+
+In the Property settings dialog you will see the "Is sensitive data" option in the bottom of the "Options" group. When this is enable the value and data in the property will only be visible to the users who have access to sensitive data.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:50]:
```suggestion
In the **Property settings** dialog, scroll to the **Is sensitive data** option at the bottom of the **Options** group. When enabled, the value and data in the property will only be visible to the users with access to sensitive data.
```

File: Reference/Security/sensitive-data.md [hunk 14]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:51]: * Click Submit.
* Click Save to save the changes.

**Review by eshanrnh — APPROVED** [review]
Looks great, @sofietoft 💪  Just a couple of suggestions

File: Reference/Security/sensitive-data.md (hunk 7 above)
Comment (sofietoft) [reply Reference/Security/sensitive-data.md:25]: Good point. I believe that step is only a thing when you have Umbraco Forms installed though 🤔 
I suggested to add another step.

File: Reference/Security/sensitive-data.md [hunk 15]
```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
...
+Any users who are not part of the Sensitive data User Group, will not be able to see the data in the properties that are marked as sensitive. Instead, they will see a generic message: "*This value is hidden. If you need access to view this value please contact your website administrator.*"
+
+![Sensitive data hidden](images/sensitive-data-hidden-v8.png)
+
+While not part of the Sensitive data User Group it is also not possible to export members or member data.
+
+Follow these steps in order to grant a user access to sensitive data:
+
+* Navigate to the **Users** section in the Umbraco backoffice.
```
Comment (sofietoft) [inline Reference/Security/sensitive-data.md:24]:
```suggestion
* Navigate to the **Users** section in the Umbraco backoffice.
* Ensure that **Users** is selected from the section tree.
```

File: Reference/Security/sensitive-data.md (hunk 11 above)
Comment (sofietoft) [reply Reference/Security/sensitive-data.md:46]: What's the reason you're removing the "double-clicking" part? 
To me, that was easier that clicking on the arrow next to the item 😅 

Also, I don't agree with emphasizing Settings again, as that is not the most important part in this step.

File: Reference/Security/sensitive-data.md (hunk 13 above)
Comment (sofietoft) [reply Reference/Security/sensitive-data.md:50]: Whether you need to scroll or not is dependent on the resolution your screen is using, right? I at least don't need to scroll to see it, when I use a bigger screen than my laptop.
So, I don't think we should mention that here, but only say that the option can be found at the bottom of the dialog!

File: Reference/Security/sensitive-data.md (hunk 14 above)
Comment (sofietoft) [reply Reference/Security/sensitive-data.md:51]: Good point! I didn't finish the step-by-step here.

I will make sure to add the final bits to the article!

**Review by sofietoft — COMMENTED** [review]
Thanks for the review @eshanrnh !

I committed most, and added a few more comments 💪

File: Reference/Security/sensitive-data.md (hunk 11 above)
Comment (eshanrnh) [reply Reference/Security/sensitive-data.md:46]: Ah ok. Double-clicking dint work for me and I had to click on the arrow to open it 😃

File: Reference/Security/sensitive-data.md (hunk 13 above)
Comment (eshanrnh) [reply Reference/Security/sensitive-data.md:50]: Yeah, agreed.

File: Reference/Security/sensitive-data.md (hunk 15 above)
Comment (eshanrnh) [reply Reference/Security/sensitive-data.md:24]: Looks perfect 🙌

File: Reference/Security/sensitive-data.md (hunk 11 above)
Comment (sofietoft) [reply Reference/Security/sensitive-data.md:46]: Strange! I'll verify!

File: Reference/Security/sensitive-data.md [hunk 16]
```diff
@@ -5,20 +5,53 @@ versionTo: 10.0.0
...
+* Ensure that **Users** is selected from the section tree.
```
Comment (eshanrnh) [inline Reference/Security/sensitive-data.md:25]:
```suggestion
* Ensure that **Users** is selected from the Users tree.
```

**Review by eshanrnh — APPROVED** [review]
Looks great, @sofietoft 🙌 Only one change is required 🙂

**Pull request merged**